README.md

```diff
@@ -111,169 +65,136 @@ After you're finished please remove all the comments and instructions!
 
 > Add a link to your deployed project.
```

Comment:
- [ ] Kindly delete the readme template text, all template text should be removed from your readme 👍🏿